modules/core/src/v2/environments.ts

```diff
@@ -63,6 +64,7 @@ const mainnetBase: EnvironmentTemplate = {
     'xpub661MyMwAqRbcEtUgu9HF8ai4ipuVKKHBzUqks4jSFypW8dwwQL1zygLgQx99NmC7zJJznSiwKG6RQfVjAKMtCsx8VjR6kQW8x7HrkXFZdnQ',
   hsmXpub:
     'xpub661MyMwAqRbcGEtTFgMAoxMFoqsRdDaiaR63byNfZdV7cBZFvovQSNJ5bpyeoQtuKVgXBk6sFQ7TtvyWqadt41DnKwveYgM5KoU2EKYjdY2',
+  batcherContractAddress: undefined, // TODO [CT-928] replace with mainnet Batcher contract address
```

Comment:
Do we know what this is yet?

Reply:
No, we don't yet. I can open up another PR to fill this in when the Batcher is deployed. Right now, the platform will block ETH batch transactions until it's deployed.